catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ValidateCommand.java

```diff
@@ -84,15 +84,17 @@
     @Reference
     CatalogFramework catalog;
 
+    protected int numMetacardsWithErrorsOrWarnings;
```

Comment:
Did we make this a `protected` field just to be able to assert on it in the unit tests? If so, we should reconsider since it breaks the class' encapsulation and doesn't really allow the unit tests to verify what really needs to be verified, i.e., that a report has been printed.

What I would suggest is keep `numMetacardsWithErrorsOrWarnings` a local variable instead of a field like it used to, make `ValidatePrinter` a non-static class, inject it in this command, mock it in the unit test and verify that `printSummary` is called with the right parameters.